include/Dollar.h

```diff
@@ -0,0 +1,12 @@
+//  Dollar.h
+#ifndef DOLLAR_H                                                                                              
+#define DOLLAR_H                                                                                              
+                                                                                                              
+class Dollar                                                                                                  
+{
+	public:                                                                                                          
```

Comment:
`public:` should have no indentation, but `Dollar()`, `~Dolar()` and any other method or member do, so please invert the situation here.